CONTRIBUTING.md

```diff
@@ -185,7 +185,7 @@ You may also test multiple notebooks this way. Passing the path to a folder will
 nox -s examples -- docs/source/examples/notebooks/models/
 ```
 
-You may also use an appropriate [glob pattern](https://www.malikbrowne.com/blog/a-beginners-guide-glob-patterns) to run all notebooks matching a particular folder or name pattern.
+You may also use an appropriate [glob pattern](https://developers.tetrascience.com/docs/common-glob-pattern) to run all notebooks matching a particular folder or name pattern.
```

Comment:
This seems like a link to another software package. Is there something more general that could be used? Like Linux documentation or python documentation?

Reply:
@kratman I have changed the link to a Linux documentation link. Is it alright? Please check